DevCycle.SDK.Server.Common/API/DVCBaseClient.cs

```diff
@@ -79,5 +79,18 @@ protected async Task<T> GetResponseAsync<T>(object body, string urlFragment, Dic
                 throw new DVCException(errorResponse);
             }
         }
+
+        public void validateEnvironmentKey(string environmentKey)
+        {
+            if (string.IsNullOrEmpty(environmentKey))
+            {
+                throw new ArgumentException("Missing environment key! Call build with a valid environment key");
+            }
+
+            if (!environmentKey.StartsWith("server") || !environmentKey.StartsWith("dvc_server"))
```

Comment:
We should probably setup a plan to stop supporting the old key format. 
Do we want to do full format validation too? Not sure how strict we want to be

Reply:
Probably not necessary since the current check will catch most cases